{"uuid": "7d0637e9-bdb0-4834-b101-a718ff6812f8", "children": ["1d6fd9ec-79f6-4596-9563-1bd3eb9f0198", "e3f554a9-2fd8-4766-b9a6-dfa51965a04d", "caa0df8c-d83b-40d6-b867-de02fe1372f3", "80e4a8d1-a655-4623-81bb-a3f9bd00b0c6", "3d0a75a7-c1f9-43ad-9ad0-d5677e5bb01b", "d338595d-ac23-46c8-9562-2a8d82a6fbef", "cecbe6eb-bb17-4ea1-856c-07f9e1a30578"], "befores": [{"name": "smodule_setup_fixture", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: chrome=99.0.4844.84)\nStacktrace:\n#0 0x55c7ecb5c7d3 <unknown>\n#1 0x55c7ec8b851f <unknown>\n#2 0x55c7ec8efd65 <unknown>\n#3 0x55c7ec8e3a19 <unknown>\n#4 0x55c7ec90c6b2 <unknown>\n#5 0x55c7ec8e32c3 <unknown>\n#6 0x55c7ec90c7be <unknown>\n#7 0x55c7ec91fa0c <unknown>\n#8 0x55c7ec90c5a3 <unknown>\n#9 0x55c7ec8e1ddc <unknown>\n#10 0x55c7ec8e2de5 <unknown>\n#11 0x55c7ecb8d49d <unknown>\n#12 0x55c7ecba660c <unknown>\n#13 0x55c7ecb8f205 <unknown>\n#14 0x55c7ecba6ee5 <unknown>\n#15 0x55c7ecb83070 <unknown>\n#16 0x55c7ecbc2488 <unknown>\n#17 0x55c7ecbc260c <unknown>\n#18 0x55c7ecbdbc6d <unknown>\n#19 0x7fc076d29609 <unknown>\n\n", "trace": "  File \"D:\\app\\Python10\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\app\\Python10\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\app\\Python10\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\售后自动化\\UIPOMTest\\project\\CRM\\test_case\\OperationMgt_PolicyAndProfits_WarrantyDurationMgt.py\", line 20, in smodule_setup_fixture\n    user.click_gotonav(\"OperationMgt\", \"PolicyandProfits\", \"WarrantyDurationMgt\")  # 点击菜单\n  File \"D:\\app\\Python10\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\售后自动化\\UIPOMTest\\project\\CRM\\page_object\\Center_Component.py\", line 20, in click_gotonav\n    self.is_click(user[level[i]])\n  File \"D:\\售后自动化\\UIPOMTest\\public\\base\\basics.py\", line 205, in is_click\n    self.find_element(locator).click()\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 81, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 710, in _execute\n    return self._parent.execute(command, params)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 425, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "前往菜单", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: chrome=99.0.4844.84)\nStacktrace:\n#0 0x55c7ecb5c7d3 <unknown>\n#1 0x55c7ec8b851f <unknown>\n#2 0x55c7ec8efd65 <unknown>\n#3 0x55c7ec8e3a19 <unknown>\n#4 0x55c7ec90c6b2 <unknown>\n#5 0x55c7ec8e32c3 <unknown>\n#6 0x55c7ec90c7be <unknown>\n#7 0x55c7ec91fa0c <unknown>\n#8 0x55c7ec90c5a3 <unknown>\n#9 0x55c7ec8e1ddc <unknown>\n#10 0x55c7ec8e2de5 <unknown>\n#11 0x55c7ecb8d49d <unknown>\n#12 0x55c7ecba660c <unknown>\n#13 0x55c7ecb8f205 <unknown>\n#14 0x55c7ecba6ee5 <unknown>\n#15 0x55c7ecb83070 <unknown>\n#16 0x55c7ecbc2488 <unknown>\n#17 0x55c7ecbc260c <unknown>\n#18 0x55c7ecbdbc6d <unknown>\n#19 0x7fc076d29609 <unknown>\n\n", "trace": "  File \"D:\\app\\Python10\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\售后自动化\\UIPOMTest\\project\\CRM\\page_object\\Center_Component.py\", line 20, in click_gotonav\n    self.is_click(user[level[i]])\n  File \"D:\\售后自动化\\UIPOMTest\\public\\base\\basics.py\", line 205, in is_click\n    self.find_element(locator).click()\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 81, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 710, in _execute\n    return self._parent.execute(command, params)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 425, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\app\\Python10\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "content", "value": "('OperationMgt', 'PolicyandProfits', 'WarrantyDurationMgt')"}], "start": 1661437132266, "stop": 1661437143150}], "start": 1661437132263, "stop": 1661437143164}], "start": 1661437132263, "stop": 1661437144260}